benchflow-experiment-manager/application/src/main/java/cloud/benchflow/experimentmanager/services/external/test/FabanManagerServiceMock.java

```diff
@@ -0,0 +1,110 @@
+package cloud.benchflow.experimentmanager.services.external.test;
+
+import cloud.benchflow.experimentmanager.constants.BenchFlowConstants;
+import cloud.benchflow.experimentmanager.services.external.FabanManagerService;
+import cloud.benchflow.experimentmanager.tasks.running.execute.ExecuteTrial.TrialStatus;
+import cloud.benchflow.faban.client.FabanClient;
+import cloud.benchflow.faban.client.exceptions.JarFileNotFoundException;
+import cloud.benchflow.faban.client.exceptions.RunIdNotFoundException;
+import cloud.benchflow.faban.client.responses.RunId;
+import cloud.benchflow.faban.client.responses.RunStatus.Code;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Jesper Findahl (jesper.findahl@gmail.com) created on 2017-06-04
+ */
+public class FabanManagerServiceMock extends FabanManagerService {
+
+  private static final String SCENARIO_ALWAYS_COMPLETED = "alwayscompleted";
```

Comment:
It will not work since we read the scenario from the `test name` which is part of the `test ID` and when converted to the Faban compatible ID the `.` are replaced with `_`. If we use it also to define the name we cannot convert back to the original ID.